scripts/build-all.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
Why do we need this script? We already have a configure.sh script?

Reply:
It was a help while making all the changes necessary for getting it to compile on Mac. Both to keep changes in one place while not all the other scripts had been updated and to have a standard way of compiling everything.

I would prefer a script like this for the CI that can be run locally to check that everything get set up correctly. Instead of the way it is now, where a fault in the CI is debugged by pushing changes to a PR.